x/posts/internal/types/msgs_test.go

```diff
@@ -115,9 +115,9 @@ func TestMsgEditPost_ValidateBasic(t *testing.T) {
 		error sdk.Error
 	}{
 		{
-			name:  "Invalid post id returns error",
+			name:  "Invalid post newID returns error",
 			msg:   types.NewMsgEditPost(types.PostID(0), "Edited post message", testOwner),
-			error: sdk.ErrUnknownRequest("Invalid post id"),
+			error: sdk.ErrUnknownRequest("Invalid post newID"),
```

Comment:
I think this is a refactoring error

```suggestion
			error: sdk.ErrUnknownRequest("Invalid post id"),
```